qa/tests/build-jts.xml

```diff
@@ -76,6 +76,7 @@
             <property name="idl.params2" value="include='${org.jboss.jbossts.qa.orb.home}/lib;${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/idl',classpath='${org.jboss.jbossts.qa.orb.classpath}'"/>
 
             <property name="is.ibmorb" value="1"/>
+			<property name="orb.jar.file.name" value="yoko-spec-corba.jar"/>
```

Comment:
Please when you revert this can you make a mention of JBTM-3600? It would be great if the build could fail with a message quoting JBTM-3600 if the JDK version is 17 and we end up in the ibmorb path.

Reply:
Okay @tomjenkinson